src/client/components/collection/CollectionsGrid.js

```diff
@@ -24,7 +24,10 @@ function CollectionsGrid({ collections, isLoading }) {
           lg={3}
           xl={3}
         >
-          <CollectionCard collection={similarCollection} isLoading={isLoading} />
+          <CollectionCard
+            collection={similarCollection}
```

Comment:
Why is this called `similarCollection`? Should be called just `collection` right?